src/main/java/edu/kis/powp/jobs2d/factories/DrawRectangle.java

```diff
@@ -0,0 +1,26 @@
+package edu.kis.powp.jobs2d.factories;
+
+import java.util.ArrayList;
+import edu.kis.powp.jobs2d.commands.ComplexCommand;
+import edu.kis.powp.jobs2d.commands.DriverCommand;
+import edu.kis.powp.jobs2d.commands.OperateToCommand;
+import edu.kis.powp.jobs2d.commands.SetPositionCommand;
+
+public class DrawRectangle implements FactoryManager{
+
+	@Override
+	public DriverCommand create() {
+		ArrayList<DriverCommand> list = new ArrayList<>();
+		list.add(new SetPositionCommand(-100, -200)); // Point 1
+		list.add(new OperateToCommand(100, -200)); //Draw line to Point 2
+		list.add(new SetPositionCommand(100, -200)); //  Point 2
```

Comment:
This and following SetPositionCommands are unnecessary! Remove. OperateToCommand already changed a position of devices head.